openassessment/xblock/static/sass/oa/utilities/_shame.scss

```diff
@@ -11,7 +11,7 @@
   // rules to overcome any poorly defined, scoped, or architected edX LMS-based CSS. Ideally this file will be removed when the LMS Front End is re-visited.
 
   // headers
-  .self-assessment__display__header, .peer-assessment__display__header, .step__header {
+  .self-assessment__display__header, .peer-assessment__display__header, .step__header, .staff-assessment__display__header {
```

Comment:
Is there any way to move these styles into the ORA2 repo? The `shame` partial was originally meant to be tentative and "shame" developers who used it :)